.pre-commit-config.yaml

```diff
@@ -0,0 +1,45 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v3.1.0
+  hooks:
+    - id: trailing-whitespace
+    - id: check-ast
+    - id: debug-statements
+    - id: end-of-file-fixer
+    - id: check-docstring-first
+    - id: check-added-large-files
+    - id: requirements-txt-fixer
+    - id: file-contents-sorter
+      files: requirements-dev.txt
+
+- repo: https://gitlab.com/pycqa/flake8
+  rev: 3.7.9
+  hooks:
+    - id: flake8
+      exclude: docs/source/conf.py
+      args: [--max-line-length=105, --ignore=E203,E501,W503, --select=select=C,E,F,W,B,B950]
+
+- repo: https://github.com/pre-commit/mirrors-isort
+  rev: v4.3.21
+  hooks:
+  - id: isort
+    additional_dependencies: [toml]
+    args: [--project=xroms, --multi-line=3, --lines-after-imports=2, --lines-between-types=1, --trailing-comma, --force-grid-wrap=0, --use-parentheses, --line-width=88]
+
+- repo: https://github.com/asottile/seed-isort-config
+  rev: v2.1.1
+  hooks:
+    - id: seed-isort-config
+
+- repo: https://github.com/psf/black
+  rev: stable
+  hooks:
+  - id: black
+    language_version: python3
+
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: v0.770
+  hooks:
+  - id: mypy
```

Comment:
I recommend all of those, including `mypy`, even for non-typed code b/c it does help to catch some inconsistencies. However, you may want to tweak the config a little bit. (There is also `black`, which is a love and/or hate style check. I learned to love it on projects with multiple contributors.)

Reply:
I'm fine using black. And I'm fine with the other additions so far as well.